Formula/brpc.rb

```diff
@@ -18,13 +19,22 @@ class Brpc < Formula
 
   depends_on "cmake" => :build
   depends_on "gflags"
+  depends_on "gperftools"
   depends_on "leveldb"
   depends_on "openssl@3"
   depends_on "protobuf@21"
```

Comment:
Would be good to leave a comment here about why this doesn't work with the latest Protobuf. (I know I tried fixing this previously, but I no longer remember the reason why it didn't work...)